### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewSearch_Keyboard: Tooltip is not visible to keyboard only users but only visible on mouse-hover.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52012]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 2.1.1 , MAS 1.1.1

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Navigate to "Match Case" , "Match Whole Word" , "Use RegEx","Use Exclude settings and Ignore Files" , "Replace All" comboboxes and "Refresh","Collapse All" and "Clear" buttons using "Tab".
4. Also, mouse hover on all above mentioned comboboxes.

**Actual:**
Tooltip information is not available for keyboard only users for "Match Case" , "Match Whole Word" , "Use RegEx","Use Exclude settings and Ignore Files","Replace All" comboboxes and "Refresh","Collapse All" and "Clear" buttons which is available on mouse hover.

**Expected:**
Tooltip information should be made available for keyboard only users which is available using mouse.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard only users will not be able to gain information regarding few controls present on screen which is only avaiable using mouse hover.

**MAS Reference:**
MAS 2.1.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

MAS 1.1.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Attachment For Reference:**
[A11y_VSCode_ViewSearch_Keyboard_Tooltip.pptx](https://github.com/Microsoft/vscode/files/2105663/A11y_VSCode_ViewSearch_Keyboard_Tooltip.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52016
